### DISCUSSION TITLE
Question about the S3DIS built-in dataset

### DISCUSSION THREAD

**Discussion by QuanticDisaster** [pyg-team/pytorch_geometric#3649] [Q&A]
Hi,

I am curious about the way S3DIS dataset is built-in in PyG, indeed each batch returned by the loaders contains 4096 points but I don't see it occuring anywhere in the source code : https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/datasets/s3dis.html
I am assuming these batchs represent 1m x 1m blocks so what should i modify if I want to use the full resolution dataset (which is around 280M points if I am not mistaken ?) with these same blocks ?

**Comment by rusty1s** [comment]
It's been some time since we have written the `S3DIS` dataset, but as far as I know, this processing of data points into mini-batches is already done in the raw data. I don't think there currently exists a way to reconstruct the full resolution dataset based on it.
